applications/adv_networking_bench/adv_networking_bench_default_tx_rx.yaml

```diff
@@ -95,4 +95,4 @@ bench_tx:
   ip_dst_addr: <5.6.7.8>  # Destination IP address - required on layer 3 network
   udp_src_port: 4096      # UDP source port
   udp_dst_port: 4096      # UDP destination port
-  address: <0000:00:00.0> # Source NIC Bus ID. Should match the address of the Tx interface above
+  address: <0000:00:00.0> # Source NIC Bus ID. Should match the address of the Tx interface above
```

Comment:
so in a way this `address` is the equivalent of `port_name` (which I think we should rename `interface`) in the Rx operator? If yes, should we do the same?

Reply:
Address was intended to be the PCIe address, linux network name, or mlx5 name of the device. the name was supposed to be a separate name to describe the interface.

Reply:
right, but the goal of this `address` field here is the same as the `port_name` field for rx, correct? which is to simply know what port/interface to send from/receive to.

Seems like we should have a consistent interface for rx and tx to define which port to interact with, either the custom name from the config, of the device system info (address, linux, mlx5). I like the former in this case, i.e. leave device system info to the advanced_network config, and the app operators just need info for headers and to know how to talk to advanced_network, so use the advanced_network interface name

Reply:
Good point. will update